Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -478,8 +478,11 @@ bool CBoot::EmulatedBS2_Wii(const DiscIO::VolumeDisc& volume,
   Memory::Write_U32(0, 0x3194);
   Memory::Write_U32(static_cast<u32>(data_partition.offset >> 2), 0x3198);
 
+  const s32 IOSOverride = Config::Get(Config::MAIN_OVERRIDE_BOOT_IOS);
+  const u64 IOS = IOSOverride != -1 ? Titles::IOS(IOSOverride) : tmd.GetIOSId();
```

Comment:
Our code style uses lower_snake_case for locals, so those should be `ios_override` and `ios` respectively.